Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -348,6 +349,56 @@ private void UpdateIndexFingerData(InteractionSourceState interactionSourceState
 #endif // WINDOWS_UWP
         }
 
+#if WINDOWS_UWP
+        // Velocity internal states
+        private float deltaTimeStart;
+        private readonly int velocityUpdateInterval = 6;
+        private int frameOn = 0;
+        private Vector3[] velocityPositionsCache = new Vector3[6];
```

Comment:
Nit: Use the same value velocityUpdateInterval instead of hardcording the number 6